doc/cephfs/cephfs-io-path.rst

```diff
@@ -0,0 +1,44 @@
+ Ceph File System IO Path
+=========================
+
+All file data in CephFS is stored as RADOS objects. CephFS clients can directly
+access RADOS to operate on file data. MDS only handles metadata operations.
+
+To read/write a CephFS file, client needs to have 'file read/write' capabilities
+for corresponding inode. If client does not have required capabilities, it sends
+a 'cap message' to MDS, telling MDS what it wants. MDS will issue capabilities
+to client when it is possible. Once client has 'file read/write' capabilities,
+it can directly access RADOS to read/write file data. File data are stored as
+Rados objects in the form of <inode number>.<object index>.
```

Comment:
`RADOS`*

Reply:
and `<inode number>.<object index>` is not completely self-explanatory, which is why I suggest talking about file layouts.